.github/workflows/node-4+.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         with:
           versionsAsRoot: true
           type: majors
-          preset: '>= 6' # preset: '>=4' # see https://github.com/import-js/eslint-plugin-import/issues/2053
+          preset: '>= 12'
```

Comment:
The following matrix can also be redredu to reflect previous changes.

Reply:
Done in [`5481192` (#11)](https://github.com/un-es/eslint-plugin-i/pull/11/commits/5481192623e2ba8bc7d32561b6d69fac6184fe03)